numpy/distutils/system_info.py

```diff
@@ -1030,20 +1027,7 @@ def calc_info(self):
 
 
 class lapack_mkl_info(mkl_info):
-
```

Comment:
What is the reason for removing this? If the class is empty, a remark explaining why would be helpful.